pandas/tests/groupby/test_rank.py

```diff
@@ -710,3 +710,12 @@ def test_rank_categorical():
 
     expected = df.astype(object).groupby("col1").rank()
     tm.assert_frame_equal(res, expected)
+
+
+def test_groupby_op_with_nullables():
+    df_ext = DataFrame({"x": [None]}, dtype="Float64")
```

Comment:
Can you add a reference to the GH issue here - the first line of the test should be `# GH 54206`